docs/EN_US/HPCCSpark/SparkHPCC.xml

```diff
@@ -438,6 +440,53 @@
       your Integrated Spark Master node's IP address.</para>
 
       <programlisting>https://192.168.56.101:8080</programlisting>
+
+      <sect2 id="Addl_Spark_Config">
+        <title>Integrated Spark Cluster Configuration Options</title>
```

Comment:
I think we should probably mention that these configuration options are in addition to the configuration options available through config manager, and are meant for edge cases and more advanced setup.

Reply:
good feedback. I will add to the doc.